idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/RemoveNullableFix.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.quickfix
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import org.jetbrains.kotlin.idea.KotlinBundle
+import org.jetbrains.kotlin.psi.*
+import org.jetbrains.kotlin.psi.psiUtil.getNonStrictParentOfType
+
+class RemoveNullableFix(element: KtNullableType, private val typeOfError: NullableKind) :
+    KotlinPsiOnlyQuickFixAction<KtNullableType>(element) {
+    enum class NullableKind(val message: String) {
+        REDUNDANT(KotlinBundle.message("remove.redundant")),
+        SUPERTYPE(KotlinBundle.message("text.remove.question")),
+        USELESS(KotlinBundle.message("remove.useless")),
+        PROPERTY(KotlinBundle.message("make.not.nullable"))
+    }
+
+    override fun getFamilyName() = KotlinBundle.message("text.remove.question")
+
+    override fun getText() = typeOfError.message
+
+    override fun invoke(project: Project, editor: Editor?, file: KtFile) {
+        val element = element ?: return
+        val type = element.innerType ?: error("No inner type " + element.text + ", should have been rejected in createFactory()")
```

Comment:
Nit: Convert error message to string template. Also it's funny how the message also mentioned "createFactory" in the previous file yet no such method exists. 😛